build/images/scripts/start_ovs_windows_containerd.ps1

```diff
@@ -0,0 +1,171 @@
+$ErrorActionPreference = "Stop"
+$OVSVersion = "3.0.5-antrea.0"
+$OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-$OVSVersion-win64.zip"
+$OVSPublishedHash = 'fd27703ef7314b26b98cffb7aea27d569530ebd3ac3c98daa981ca2654373032'
+$WorkDir = [System.IO.Path]::GetDirectoryName($myInvocation.MyCommand.Definition)
+$OVSDownloadDir = $WorkDir
+$OVSInstallDir = "C:\openvswitch"
+$OVSZip = "$OVSDownloadDir\ovs-win64.zip"
+$ImportCertificate = $true
+$CheckFileHash = $true
+$PowerShellModuleBase = "C:\Windows\System32\WindowsPowerShell\v1.0\Modules"
+$InstallLog = "$OVSDownloadDir\install_ovs.log"
+
+function WaitExpandFiles($Src, $Dest) {
+    Write-Host "Extract $Src to $Dest"
+    Expand-Archive -Path $Src -DestinationPath $Dest | Out-Null
+}
+
+function CheckIfOVSInstalled() {
+    if (Test-Path -Path $OVSInstallDir) {
+        Write-Host "$OVSInstallDir already exists, OVS is already installed."
+        return $true
+    }
+    else {
+        Write-Host "$OVSInstallDir does not exist, OVS is not installed."
+        return $false
+    }
+}
+
+function CreatePath($Path){
+    if ($(Test-Path $Path)) {
+        mv $Path $($Path + "_bak")
+    }
+    mkdir -p $Path | Out-Null
+}
+
+function DownloadOVS() {
```

Comment:
Why do we need such functions, isn't openvswitch files exising in windows-ovs image?

Reply:
I mean we don't need to download ovs again As we are using image antrea/antrea-windows, which already include the downloaded and unzipped openvswitch files. You copy directly if you want, although I don't feel it is necessary to do so.

Reply:
@NamanAg30 , openvswitch folder includes driver files and it is part of windows-ovs image and antrea-windows image, you can use it from there. Downloading again is not required here.

Reply:
We do not need the installation steps after new changes